test/smoke_test/bcda_client.go

```diff
@@ -44,46 +40,51 @@ func init() {
 	flag.StringVar(&apiVersion, "apiVersion", "v1", "resourceType to test")
 	flag.IntVar(&timeout, "timeout", 300, "amount of time to wait for file to be ready and downloaded.")
 	flag.StringVar(&endpoint, "endpoint", "", "base type of request endpoint in the format of Patient or Group/all or Group/new")
+	flag.IntVar(&httpRetry, "httpRetry", 3, "amount of times to retry an http request")
```

Comment:
could this potentially be an environmental config?

Reply:
It can be, but since bcda_client.go is an executable (with the presence of the func main), we're able to leverage the [flags package](https://golang.org/pkg/flag/) to supply these config options instead of using environment variables.

When we invoke the bcda_client via `go run`, we can supply the `httpRetry` flag whose value is controlled by an external environment variable. There's some examples of this in the [bulk_data_requests.sh](https://github.com/CMSgov/bcda-app/blob/master/test/smoke_test/bulk_data_requests.sh#L7).

Reply:
ok, makes sense